flight/mcu/lib/ServoControl/ServoControl.cpp

```diff
@@ -47,13 +47,14 @@ void ServoControl::servoActuation(float length) {
     // The angle is found through utilizing a fft and mapping extension/angle
     // values to a sine function. len (mm), pass in ang (rad)
 
-    if (length < 0) length = 0;
-    if (length > 0.018) length = 0.018;
+    // if (length < 0) length = 0;
+    // if (length > 0.018) length = 0.018;
```

Comment:
Do we not want to limit the length?

Reply:
Round off angle takes care of this

Reply:
No matter what angle we write to the servos, we restrict it to between 0 and 130

Reply:
Never mind, we put the length limitations back. This is because the function that maps lengths to angles is an arcsin, for which some inputs are undefined. This means that for some length inputs, it returns an undefined angle (extremely bad if that happened during launch. )